heftybyte/challenge-github-user-list-blueyedgeek
# Dev

### What does this PR do

It implements the required functionality as specified in the readme file

### How to manually test

Clone the `dev` branch, run `yarn` to install all dependencies and then `yarn start` to start the development server. The application should be running on `localhost:3000`

### Image(s)

<img width="1278" alt="screen shot 2017-08-17 at 12 38 17 am" src="https://user-images.githubusercontent.com/7131935/29389813-6ceaa40e-82e4-11e7-8ac7-41e1cca6524b.png">
